### ISSUE TITLE
Little error in sample codes....

### ISSUE THREAD

**Issue by objects76** [qicosmos/rest_rpc#3]
First.... thank you for your great/simple rpc lib.

The following codes in examples\app_client\client.hpp has error, right?
`Maybe ... client_.async_call<FUTURE>("add", a,b) ?`

	std::future<req_result> async_add(int a, int b) {
		return client_.async_call("add", a, b);
	}

**Comment by qicosmos** [comment]
yeah, you are right, i didn't update the app_client to the latest code, i will do the update soon.

**Comment by qicosmos** [comment]
I have fixed the compile error
